content/projects/gsoc/2022/project-ideas/plugin-health-scoring-system.adoc

```diff
@@ -20,18 +20,76 @@ Plugin health scoring to help maintainers, new contributors, and Jenkins users.
 
 === Background
 
-Keeping a plugin current with the ongoing development of Jenkins can require user interface improvements, dependency updates, and more.
+Keeping a plugin current with the ongoing development of Jenkins is key for a smooth operation.
+Keeping it current can require user interface improvements, dependency updates, and more.
+
+Measurements of the plugin's "health" will help **maintainers** as they decide how to invest their development time.
+With such a system. items that need to be maintained, improved, or adapted are identified. +
+For example, having Release Drafter correctly configured and Pull Request labeling in place is a recognized good practice. 
+It reduce the effort to provide a changelog with each release. +
+Dependabot properly configured will reduce the dependency update effort by automatically submitting update pull request.
 
-Measurements of the "health" of a plugin will help maintainers as they decide how to invest their development time.
-Measurements of the "health" of a plugin will help users as they decide if they should install and use a plugin.
+Measurements of the plugin's "health" will help **users** to decide if they should install and use a plugin. 
+Plugins with a lower health score maybe less likely to meet the user's reliability needs.
+It is similar to a badly maintained garden that will make harvesting difficult and not yield good vegetables or fruits. 
 
 === Quick Start
 
+A presentation of the "Plugin Health Score" project was made during the 10-Feb-2022 GSoC Office Hour.
+
+// video::https://youtu.be/i7Y0FM1tms4[youtube,width=800,height=420,start=nnn]
+video::https://www.youtube.com/watch?v=i7Y0FM1tms4[youtube,width=800,height=420,start=488]
```

Comment:
No need to embed the YouTube URL when the type is set to YouTube inside the brackets
```suggestion
video::i7Y0FM1tms4[youtube,width=800,height=420,start=488]
```